src/frontend/src/lib/components/send/SendForm.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import SendSource from '$lib/components/send/SendSource.svelte';
+	import ButtonGroup from '$lib/components/ui/ButtonGroup.svelte';
+	import ContentWithToolbar from '$lib/components/ui/ContentWithToolbar.svelte';
+	import { balance } from '$lib/derived/balances.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { token } from '$lib/stores/token.store';
+
+	export let source: string;
+
+	const dispatch = createEventDispatcher();
+</script>
+
+<form on:submit={() => dispatch('icNext')} method="POST">
+	<ContentWithToolbar>
+		<slot name="destination" />
+
+		<slot name="amount" />
+
+		<SendSource token={$token} balance={$balance} {source} />
```

Comment:
my suggestion is to pass these things as external props, so that we can re-use it.

I agree that at some point these should be "internal". I am working on that with the send context, but still early

Reply:
Agreed, updated.